samples/net/sockets/socket_dumb_http/Makefile

```diff
@@ -0,0 +1,23 @@
+# Makefile - dumb socket-based http server
+
+#
+# Copyright (c) 2017 Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+BOARD ?= qemu_x86
+CONF_FILE ?= prj.conf
+
+include $(ZEPHYR_BASE)/Makefile.inc
+
+ifeq ($(CONFIG_NET_L2_BLUETOOTH), y)
+	QEMU_EXTRA_FLAGS = -serial unix:/tmp/bt-server-bredr
+else
+	include $(ZEPHYR_BASE)/samples/net/common/Makefile.ipstack
+endif
+
+src/socket_dumb_http.o: src/response.inc
+
+response.inc: response.html
+	bin2h 8 <$^ >$@
```

Comment:
This requires that you have bin2h somewhere. At least in my host (Fedora 25) I do not have it.
One option could be that I send the patch "samples: net: common: Add rules to generate a header file" (currently found in websocket patchset) separately as there could be some use for it here.

Reply:
Those lines are actually artifact of experimentation, which didn't work still. Instead, I included pre-generated *.inc files into this PR (and standalone Makefile.gen file to re-generate them). But indeed, this:

> that I send the patch "samples: net: common: Add rules to generate a header file" (currently found in websocket patchset) separately as there could be some use for it here.

would be better, I'd appreciate it.

Reply:
I created a separate PR #1447 that contains one way to create an include file.